docs/ansible/role-index.rst

```diff
@@ -118,6 +118,7 @@ Databases
 - :ref:`debops.redis_server`
 - :ref:`debops.redis_sentinel`
 - :ref:`debops.slapd`
+- :ref:`debops.influxdb`
 - ``debops.phpmyadmin``
```

Comment:
influxdb_server needs to be added as well.

Reply:
Yes. Today I will upload documentation for debops.influxdb_server role.